features/facebookImport/src/locales/en/miniStories/advertisingValue.json

```diff
@@ -1,5 +1,5 @@
 {
     "title": "Your advertising value",
-    "intro": "Since Facebook makes its money by selling advertising they assigned <strong>{{number_interests}}</strong> so-called advertising interests to you, for example",
+    "intro": "Since Facebook makes its money by selling advertising they assigned <strong>{{number_interests}}</strong> so-called advertising interests to you. For example, Facebook assumes you are interested in:",
```

Comment:
alt: they *have assigned* | they *assign*
alt also: ", for example" → such as. Avoids repetition of Facebook, as well as the word interest.

Reply:
Also, no equivalent in German? I miss the chance to show my ignorance of that language...

Reply:
Oh I just copy pasted the text from here https://wiki.polypoly.eu/display/POLYPOD/Text+for+Teasers+and+Tech+Demos+on+Importer+Overview+Screen
The german texts will be delivered by Christian later